bors.toml

```diff
@@ -1,7 +1,13 @@
 status = [
-  "test-os (ubuntu-latest)",
-  "test-os (windows-latest)",
-  "test-os (macos-latest)",
+  "test-os 1.4 - (ubuntu-latest)",
```

Comment:
all of this seems completely unnecessary tbh

Reply:
Ok, should I just add 1.5 and remove 1.4?

Reply:
yes, let's focus energies on 1 well supported version at a time